src/components/UserSelector.tsx

```diff
@@ -1,45 +1,33 @@
-import React, { useContext, useEffect, useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import classNames from 'classnames';
-import { UserContext } from './UsersContext';
-import { User } from '../types/User';
+import * as authorActions from '../features/author';
+import * as usersActions from '../features/users';
```

Comment:
Consider renaming the import to better reflect its usage, such as `initUsersActions`, to avoid confusion with other actions.